lib/cfn-nag/util/enforce_dynamic_reference.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Returns false if the provided key_to_check is a dynamic reference
+# to SSM Secure or Secrets Manager; true otherwise.
+# Only applicable for a string
+def insecure_dynamic_reference?(_cfn_model, key_to_check)
```

Comment:
i think this logic is correct - but the name of the method seems to state that it is only checking dynamic references.... perhaps a better name is insecure_literal_or_dynamic_reference?()

Reply:
naming has been changed